pkg/cmd/util/output/backup_describer.go

```diff
@@ -680,6 +680,7 @@ func describeDataMovement(d *Describer, details bool, info *volume.BackupVolumeI
 		}
 		d.Printf("\t\t\t\tData Mover: %s\n", dataMover)
 		d.Printf("\t\t\t\tUploader Type: %s\n", info.SnapshotDataMovementInfo.UploaderType)
+		d.Printf("\t\t\t\tMoved data Size (bytes): %d\n", info.SnapshotDataMovementInfo.Size)
```

Comment:
Was this related to the namespace selector enhancement?

Reply:
No. This is used to display the added information from PR #7554.
I forgot to modify the CLI output in that PR.